tracks/scala/exercises/two-fer/mentoring.md

```diff
@@ -0,0 +1,12 @@
+### Reasonable solutions
+
+```scala
+object Twofer {
+  def twofer(name: String = "you"): String =
+    s"One for $name, one for me."
+}
+```
+
+### Common suggestions
+
+- Use default parameters instead of multipe `twofer` functions.
```

Comment:
I don't know if this happens a lot, but for C# a lot of people construct the dynamic string without using interpolation (using `+` or `String.format`). If this is common in Scala, it might be useful to also add a comment about this too.

Reply:
Good point! I have stole some text from the C# notes, and tweaked it for the Scala nodes.
Thanks